From: Jeremy Bicha <jeremy.bicha@canonical.com>
Date: Thu, 1 Sep 2022 10:05:28 -0400
Subject: nautilus: Don't import a specific version

This isn't needed since
https://gitlab.gnome.org/GNOME/nautilus-python/-/commit/8c88de8da42

And interferes with being able to use this extension
on Nautilus 43+
---
 data/nautilus/open-tilix.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/nautilus/open-tilix.py b/data/nautilus/open-tilix.py
index 16a9b97..21a1223 100644
--- a/data/nautilus/open-tilix.py
+++ b/data/nautilus/open-tilix.py
@@ -14,7 +14,6 @@ except ImportError:
 
 from gi import require_version
 require_version('Gtk', '3.0')
-require_version('Nautilus', '3.0')
 from gi.repository import Gio, GObject, Gtk, Nautilus
 
 

